Resources/Computing/SSHComputingElement.py

```diff
@@ -356,14 +357,14 @@ def _prepareRemoteHost(self, host=None):
     self.log.verbose('Creating working directories on %s' % self.ceParameters['SSHHost'])
     result = ssh.sshCall(30, cmd)
     if not result['OK']:
-      self.log.warn('Failed creating working directories: %s' % result['Message'][1])
+      self.log.error('Failed creating working directories: %s' % result['Message'][1])
       return result
     status, output, _error = result['Value']
     if status == -1:
-      self.log.warn('Timeout while creating directories')
+      self.log.error('Timeout while creating directories')
       return S_ERROR('Timeout while creating directories')
     if "cannot" in output:
-      self.log.warn('Failed to create directories: %s' % output)
+      self.log.error('Failed to create directories: %s' % output)
```

Comment:
```suggestion
      self.log.error('Failed to create directories', '(%s)' % output)
```

And for this one, and other "standard", please use `errno` or `DErrno` codes.